assets/scss/theme.scss

```diff
@@ -1,5 +1,5 @@
-$font-family-monospace: Hack, Consolas, Monaco, "Andale Mono", monospace !default;
-$font-family-sans-serif: "Open Sans", Helvetica, Arial, sans-serif !default;
+$font-family-monospace: hack, consolas, monaco, "Andale Mono", monospace !default;
```

Comment:
i think it would be better to leave them capitalized, since we also use Hack as capitalized in our font-face declaration. i think the common ground could be to quote the font-face names and only leave the builtin keywords unquoted.

Reply:
✅